adding_species_manuscript.tex

```diff
@@ -634,114 +641,112 @@ \subsection*{Extracting parameters from the literature}
-nor genetic maps.
-Finally, although we could crowd-source addition of many species,
-still each one required substantial attention by a core group of maintainers.
-So, the benefit of including such species in \stdpopsim would be outweighed
-by the substantial extra burden on downstream users and \stdpopsim maintainers.
-
-However, simulation is still useful in such species.
+%ILAN: switched location of table, and edited its contents.
+% I tried to focus the last column on features that could end up being screwed
+% up by a faulty choice of parameter value. I hope it's clear enough.
+\begin{table}[b!]
+	\captionof{table}{\textbf{Guidelines for dealing with missing parameters.} %
+		For each parameter, we provide a suggested course of action, 
+		and mention the main discrepancies between the simulated data
+		and real genomic data,
+		which can be caused by mis-specification of that parameter.
+	} \label{tab:param-mod}
+	\begin{tabular}{p{1.5in}p{2.2in}p{2.2in}}
+		\hline
+		\textbf{Missing parameter}  & 
+		\textbf{Suggested action} & 
+		\textbf{Possible discrepancies} \\
+		\hline
+		Mutation rate      &
+		Borrow from closest relative with a citeable mutation rate &
+		Number of polymorphic sites  \\
+		\hline
+		Recombination rate &
+		Borrow from closest relative with a citeable recombination rate &
+		Patterns of linkage disequilibrium
+		\\
+		\hline
+		Gene conversion rate and tract length &
+		Set rate to 0 or borrow from closest relative with a citeable rate &
+		Lengths of shared haplotypes across individuals
+		\\
+		\hline
+		Demographic model &
+		Set the effective population size (Ne) to a value
+		that reflects the average observed genetic diversity in the
+		simulated population
+		     &
+		Features of genetic diversity that are captured by the site frequency spectrum,
+		such as the prevalence of low-frequency alleles\\
+		\hline
+	\end{tabular}
+\end{table}
+
+%ILAN: following suggestions from others, I moved most of this into the table,
+% and added a clearer description of the table in the main text.
+For many species it is difficult to obtain estimates of all necessary model parameters.
+Table \ref{tab:param-mod} provide suggestions for ways to deal with missing values of various central model parameters.
+The table also mentions the main discrepancies between the simulated data and real genomic data,
+which can be caused by mis-specification of each parameter.
+
+%ILAN: shortened the paragraph by removing a lot of the low-level considerations,
+%  following a suggestion by Jerome
+Several researchers who participated in the ``Growing the Zoo'' hackathon wished to add species
+whose genome assemblies are composed of many relatively small contigs,
+unanchored to chromosome-level scaffolds.
+Although we wish to keep \stdpopsim as inclusive as possible,
+we made a conscience decision to only add species with chromosome-level assemblies.
```

Comment:
It's not so clear what 'inclusive' means here (like, we also aim to be an inclusive community of people working together); how about
```suggestion
Although we had not previously put restrictions on which species might be added,
we decided that we would only add species with chromosome-level assemblies.
```

Reply:
accepted your suggestion and will include in the next PR